_episodes/03-use-registry.md

```diff
@@ -78,25 +101,21 @@ A standard can be defined as "a structure agreed and adopted by a community" or
 >
 {: .callout}
 
-TODO: difference between control vocabulary and ontology.
-
+**Controlled vocabularies** provide a way to organize knowledge for subsequent retrieval. It is usually a carefully selected list of words and phrases, which are used to tag units of information (document or work) so that they may be more easily retrieved by a search. The fundamental difference between an **ontology** and a **controlled vocabulary** is the level of abstraction and relationships among concept. A formal ontology is a controlled vocabulary expressed in an ontology representation language. ([ref](https://semwebtec.wordpress.com/2010/11/23/contolled-vocabulary-vs-ontology/))
```

Comment:
I left this unresolved as I think it may be too technical. Thoughts?

Reply:
It would be indeed too technical. What about "The fundamental difference between and **ontology** and other **controlled vocabularies**, e.g., thesauri, is the [...]"?

Reply:
That is a great point! Fixed in a new commit.